src/typing_manager.ts

```diff
@@ -188,12 +195,32 @@ export class TypingManager implements Disposable {
         this.registerReplacePrevChar();
         await this.onSendCommand(key);
     };
+    public onDisableCommand = (): void => {
+        this.neovimToggle = true;
+        this.client.command("stopinsert");
+    };
+
+    public onEnableCommand = (): void => {
```

Comment:
this could take an `enable: 'enable'|'disable'|'toggle'` param, then all the commands could use a single function.

Reply:
Changed as requested